contribs/gmf/apps/mobile/index.html

```diff
@@ -30,6 +30,10 @@
         ng-click="mainCtrl.toggleRightNavVisibility()">
       </button>
       <div class="overlay" ng-click="mainCtrl.hideNav()"></div>
+      <button class="ngeo-mobile-geolocation" ngeo-mobile-geolocation="::"
```

Comment:
The `class="ngeo-mobile-geolocation"` is rdundant.
You can use `[ngeo-mobile-geolocation]` pattern to declare a css class on this element.